apps/sharedtest/test/unit/gzip_worker_test.js

```diff
@@ -0,0 +1,42 @@
+/* global Gzip */
+'use strict';
+
+require('/shared/js/gzip/gzip.js');
+
+suite('gzip worker', function() {
+
+  test('worker successfully compresses payload', function(done) {
+    var payload = 'abcdefghijklmnopqrstuvwxyz1234567890';
+
+    // The first ten bytes are the header of the compressed data. Part of the
+    // header will vary each time the test is run (timestamp); parts of the
+    // header will vary based on the OS on which the test is running. As such,
+    // the test will compare the fixed values of the header that are
+    // significant to gzip compressed data.
+    //
+    // Bytes 0 and 1 should be: 31, 139 -- 'gzip' format
+    // Byte 3 should be 8 -- 'inflate' compression algorithm
+    var expectedCompressedData =
+      new Uint8Array([
+        31,139,8,0,58,51,24,86,0,255,5,0,5,17,128,48,176,18,38,35,14,48,119,
+        183,244,127,239,247,99,66,25,23,82,105,99,157,15,49,229,82,91,31,115,
+        221,246,227,188,110,244,44,0,88,155,95,95,36,0,0,0]);
+
+    Gzip.WORKER_URL = '/' + Gzip.WORKER_URL;
```

Comment:
If the / is all you need to add here, maybe you can hardcode the slash it in gzip.js and not bother with this WORKER_URL stuff?